lib/Catalyst/Plugin/Sentry.pm

```diff
@@ -0,0 +1,167 @@
+package Catalyst::Plugin::Sentry;
+
+use v5.10;
+use strict;
+use warnings;
+
+use DBDefs;
+use Devel::StackTrace;
+use File::Slurp qw( read_file );
+use IO::File;
+use Moose;
+use Sentry::Raven;
+
+our $suppress = 0;
+
+sub execute {
```

Comment:
Why it's a pain to use Sentry::Raven::capture_errors with Catalyst:

* It doesn't expose the captured error, so we can't e.g. check its class to
suppress timeouts.
* Using `next::method` within a callback doesn't do what is intended. While we
could ditch plugins and do error handling within `dispatch`, we still have
the first problem.